packages/@react-aria/dnd/test/useDroppableCollection.test.js

```diff
@@ -212,11 +212,15 @@ describe('useDroppableCollection', () => {
         ]
       });
 
-      await waitFor(() => expect(within(grid).getAllByRole('gridcell')).toHaveLength(4));
+      await waitFor(() => expect(within(grid).getAllByRole('gridcell')).toHaveLength(4), {interval: 10});
```

Comment:
why 10? i don't see anything that advances timers by 10, also, how does this waitFor even work with faked timers like this? wouldn't it never check again after the first time (@t=0) no matter what interval you set?

Reply:
I believe `waitFor` calls `advanceTimers` repeatedly on the given interval. I don't remember exactly, but the problem as I remember was that the default interval was jumping too far ahead, past our 50ms timer for the drop, so I lowered it.

Reply:
well, mind blown, thanks for the the explanation